dev/tests/integration/testsuite/Magento/Catalog/_files/multiselect_attribute.php

```diff
@@ -34,21 +43,11 @@
             'used_in_product_listing' => 0,
             'used_for_sort_by' => 0,
             'frontend_label' => ['Multiselect Attribute'],
-            'backend_type' => 'varchar',
+            'backend_type' => 'text',
             'backend_model' => \Magento\Eav\Model\Entity\Attribute\Backend\ArrayBackend::class,
             'option' => [
-                'value' => [
-                    'option_1' => ['Option 1'],
-                    'option_2' => ['Option 2'],
-                    'option_3' => ['Option 3'],
-                    'option_4' => ['Option 4 "!@#$%^&*']
-                ],
-                'order' => [
-                    'option_1' => 1,
-                    'option_2' => 2,
-                    'option_3' => 3,
-                    'option_4' => 4,
-                ],
+                'value' => $valueOptionArray,
```

Comment:
Please revert changing of the values. Instead, please just copy-paste this file and the one with the _rollback.php suffix and adjust it how you need it.

Reply:
I've modified fixtures to work with 2 multiselect attributes, you can mark this as resolved